config/samples/sw.kogito_v1alpha08_kogitoserverlessworkflow_devmodeWithExternalResource.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: sw.kogito.kie.org/v1alpha08
+kind: KogitoServerlessWorkflow
+metadata:
+  name: greeting
+  annotations:
+    sw.kogito.kie.org/description: Greeting example on k8s!
+    sw.kogito.kie.org/version: 0.0.1
+    sw.kogito.kie.org/profile: dev
+    sw.kogito.kie.org/resource-camel: mycamel-configmap
```

Comment:
add the camel route example well, ideally a fully working example integrated with this CR.

Reply:
I can add a yaml with the configmap but the configmap must be created manually by the user, can't be integrated with this CR

Reply:
ok

Reply:
cool, maybe add in the instructions how to deploy and use?